### ISSUE TITLE
May I know how to rmsprop init from sgd?

### ISSUE THREAD

**Issue by billhhh** [marvis/pytorch-mobilenet#17]
Thank you for sharing your idea, while I encounter the same problem as you.
The loss did not go down when I train models with rmsprop on imagenet.

You mentioned
rmsprop init from sgd : top1 69.526 top5 88.978

So could you share your Hyper Params: batchsize ; lr of both init from sgd and from rmsprop; did you init from sgd from 1st epoch? if not, how many epoches?

Really appreciate your help!

**Comment by ranery** [comment]
Same problem, wait for your suggestion!

**Comment by notabigfish** [comment]
Did anyone figure this out? Thanks!
